build/package/Dockerfile.start

```diff
@@ -1,10 +1,13 @@
-FROM registry.access.redhat.com/ubi8
+FROM registry.access.redhat.com/ubi8/go-toolset:latest
```

Comment:
Using this as a base allows for better reuse of layers. However not really sure if this is the final way to go.